src/main/java/com/ericsson/ei/controller/model/Subscription.java

```diff
@@ -195,12 +208,12 @@ public void setAdditionalProperty(String name, Object value) {
 
     @Override
     public String toString() {
-        return new ToStringBuilder(this).append("aggregationtype", aggregationtype).append("created", created).append("notificationMeta", notificationMeta).append("notificationType", notificationType).append("restPostBodyMediaType", restPostBodyMediaType).append("notificationMessageKeyValues", notificationMessageKeyValues).append("repeat", repeat).append("requirements", requirements).append("subscriptionName", subscriptionName).append("userName", userName).append("password", password).append("ldapUserName", ldapUserName).append("additionalProperties", additionalProperties).toString();
+        return new ToStringBuilder(this).append("aggregationtype", aggregationtype).append("created", created).append("notificationMeta", notificationMeta).append("notificationType", notificationType).append("restPostBodyMediaType", restPostBodyMediaType).append("notificationMessageKeyValues", notificationMessageKeyValues).append("repeat", repeat).append("requirements", requirements).append("subscriptionName", subscriptionName).append("authenticationType", authenticationType).append("userName", userName).append("password", password).append("ldapUserName", ldapUserName).append("additionalProperties", additionalProperties).toString();
```

Comment:
Could we have each append on its own line?

Reply:
This is a generated class.